icu4j/main/tests/core/src/com/ibm/icu/dev/test/lang/UCharacterCaseTest.java

```diff
@@ -467,6 +470,72 @@ public void TestDutchTitle() {
         assertEquals("Dutch titlecase check in Dutch with nolowercase option",
```

Comment:
FYI -- I looked for the C/C++ test function equivalent to this. It looks like there was no C++ version, so it's good to just port it, and intltest/strcase.cpp is right for that.

What we do have is a C (cintltst) function, but it would be unnecessarily inconvenient to extend *that* with the new cases. https://github.com/unicode-org/icu/blob/main/icu4c/source/test/cintltst/cstrcase.c#L334

And we also have a data-driven test, but that is also hard for specific API coverage: https://github.com/unicode-org/icu/blob/main/icu4c/source/test/testdata/casing.txt#L46

Reply:
The new test in intltest/strcase.cpp passes now, after converting the C++ UTF-16 code to the UTF-8 version, because the strcase.cpp tests call both versions.

But now it seems like these fixes have triggered a different failure in a UnicodeString test now where it is calling title case methods.